src/datasets/commands/dummy_data.py

```diff
@@ -112,8 +112,9 @@ def _create_dummy_data(
         encoding = encoding or DEFAULT_ENCODING
         if os.path.isfile(src_path):
             logger.debug(f"Trying to generate dummy data file {dst_path}")
+            dst_path_extension = get_main_extension(dst_path)
```

Comment:
I think you can simply use pathlib here
```python
import pathlib

print(pathlib.Path('yourPath.example').suffix) # '.example'
```

Reply:
@lhoestq yes, you are right!! Not exactly .suffix, but .suffixes... I am changing this. ;)

Reply:
@lhoestq note that in the case "file.xml.gz" I want to consider the first extension ".xml" and not the last one.